src/test/java/seedu/address/testutil/TypicalTasks.java

```diff
@@ -39,8 +39,9 @@
     public static final Task CS2103T_PROJECT = new TaskBuilder().withName("CS2103T Project")
             .withDescription("V1.2b completion").withDeadline("2022-03-13").withCompletionStatus("false")
             .withTags("CS2103T").build();
+    /** Task with Completion Status set to true **/
     public static final Task MEET_ALICE = new TaskBuilder().withName("Meet Alice")
-            .withDescription("Meeting Alice at NUS").withDeadline("2022-04-01").withCompletionStatus("false").build();
+            .withDescription("Meeting Alice at NUS").withDeadline("2022-04-01").withCompletionStatus("true").build();
```

Comment:
Likewise, should you update this to be a `Task` instead of a `Person?
Otherwise, I have shelved this issue in https://github.com/AY2122S2-CS2103T-T09-1/tp/issues/152!